go-backend/db/organization.go

```diff
@@ -0,0 +1,216 @@
+package db
+
+import (
+	"context"
+	"time"
+	"regexp"
+	logger "github.com/sirupsen/logrus"
+)
+
+const (
+	CreateOrganizationQuery = `INSERT INTO organizations (
```

Comment:
is there a reason to make these variables exported, I don't see they are being used from outside the package.